README.md

```diff
@@ -57,32 +59,55 @@ gem install bundler
 
 ```
 git clone git@github.com:ros2/rosindex.git --recursive
-cd rosindex.github.io
+cd rosindex
 bundle install
 ```
 
-### Clone repos that are part of rosdistro and build the index
+#### Clone repos that are part of rosdistro and build the index
 
-```
+Run:
+
+```sh
 make build
 ```
 
-### Serve the devel (tiny) version locally
+By default, site will be written to _site. This behavior can be
+overriden as follows:
 
+```sh
+make build site_path=/path/to/site
 ```
-make serve-devel
+
+### On the provided Ubuntu 16.04 container
+
+#### Pre-requisites
+
+##### Docker
+
+See https://docs.docker.com/install/linux/ for details on docker installation.
+
+#### Build docker image
+
+```sh
+docker/build.sh
```

Comment:
`sh` fails for me with the following error, but `bash` works. Maybe use `bash` on the instructions?

```
$ sh docker/build.sh build_site
readlink: invalid option -- 'p'
Try 'readlink --help' for more information.
dirname: missing operand
Try 'dirname --help' for more information.
unable to prepare context: unable to evaluate symlinks in Dockerfile path: lstat /image: no such file or directory
```

Reply:
Totally right. Documentation highlights are wrong. Fixed.